sympy/polys/tests/test_polytools.py

```diff
@@ -2282,7 +2282,6 @@ def test_sqf():
     raises(ComputationFailed, lambda: sqf_part(4))
 
     assert sqf(1) == 1
-    assert sqf_list(1) == (1, [])
```

Comment:
Why was this removed?

Reply:
I wasn't sure if error should be printed or given output should be given out. It's clear now.